lab-doug/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+var myGreet = require(__dirname + '/lib/greet');
```

Comment:
i usualy name the variables that reference my modules after the name of their file,,, that way when the code is complex its easy to tell where things come from